.github/workflows/functional-test.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: v1.21.5
```

Comment:
Can you use 1.19 version as is in https://github.com/project-codeflare/instascale/blob/main/.github/workflows/go.yml#L22 ?

Reply:
Yes, I will update it.